how-to/how-to-use-rootless-kata-containers-with-podman.md

```diff
@@ -0,0 +1,171 @@
+
+```bash
+$ [ -f /etc/selinux/config ] && sudo sed -i 's/^SELINUX=.*/SELINUX=disabled/g' /etc/selinux/config
+```
+
+#### 2. Add user to KVM group
+If running a KVM based hypervisor, the user running the workload needs to be added to the KVM group:
+```bash
+$ sudo usermod -a -G kvm $USER
+```
+
+#### 3. Reboot
+Reboot the system for the changes to take effect (a reboot is required when disabling SELinux, while logging out and back in is enough to have that user joining the `KVM` group).
+
+You can now verify if the configuration is correct:
+
+* (if installed) SELinux should have been disabled:
+```bash
+$ getenforce
+Disabled
+```
+
+* The user should be in the `kvm` group:
+```
+$ groups | grep -ow kvm
+kvm
+```
+
+#### 4. Setup Kata configuration files
+If the Kata `configuration.toml` file does not exist in `/etc`, do the following:
+```bash
+$ sudo install -D -o ${USER} -g root -m 0640 /usr/share/defaults/kata-containers/configuration.toml /etc/kata-containers
+```
+
+Or, if the file exists, but is not readable by the user:
+```bash
+$ sudo chown -R a+r /etc/kata-containers/
+```
+
+#### 5. Disable `vhost-net`
+Disable `vhost-net` in the Kata configuration file, by commenting out the `disable_vhost_net` line:
+```bash
+$ sudo sed -i -e 's/^#disable_vhost_net = true/disable_vhost_net = true/' /etc/kata-containers/configuration.toml
+```
+
+#### 6. Modify the Kata images permissions
+The Kata images needs to be accessible by the user, so change the permissions of the image directory to be readable by the user.
+```bash
+$ sudo chown -R a+r /usr/share/kata-containers
+```
+
+#### 7. Set up Podman rootless configuration
+If `libpod.conf` does not exist in `~/.config/containers/`:
+```bash
+$ sudo install -D -o ${USER} -g ${USER} -m 0640 /usr/share/containers/libpod.conf ~/.config/containers/
+```
+
+By default the `tmp_dir` in `libpod.conf` is set to `/var/run/libpod`, however that is not accessible by a user, so change to the rootless runtime directory.
+```bash
+$ sed -i -e "s|^tmp_dir = .*$|tmp_dir = \"$XDG_RUNTIME_DIR/libpod/tmp\"|" ~/.config/containers/libpod.conf
+```
+
+#### 8. Add Kata Runtime to Podman configuration file (optional)
+You can tell Podman to create or run containers using the Kata runtime with `--runtime=value` flag.
+
+You can directly pass the fully qualified Kata runtime path with:
+```bash
+$ podman run --runtime=/usr/local/bin/kata-runtime ...
+```
+
+or add a `kata` entry in the `[runtimes]` section of the configuration file by appending the Kata Runtime binary path(s) to the `libpod.conf` file:
+```bash
+$ echo 'kata = ["/usr/local/bin/kata-runtime"]' >> ~/.config/containers/libpod.conf
+```
+and then use `kata` as the runtime name:
+```bash
+$ podman run --runtime=kata ...
+```
+
+> **NOTE:**
+> A less recommended approach could be to have the absolute `kata-runtime` path in the standard `$PATH` location instead of the configuration file, and a binary with that name will be looked up automatically:
```

Comment:
Suggested rewrite: 
… location instead of the configuration file. In this case it looks up a binary with that name automatically: